x/mongo/driver/errors.go

```diff
@@ -230,8 +230,27 @@ func (e Error) Retryable() bool {
 			return true
 		}
 	}
-	if strings.Contains(e.Message, "not master") || strings.Contains(e.Message, "node is recovering") {
-		return true
+
+	return false
+}
+
+// RetryableWrite returns true if the error is retryable for a write operation
+func (e Error) RetryableWrite(wireVersion *description.VersionRange) bool {
+	for _, label := range e.Labels {
+		if label == NetworkError {
+			return true
+		}
+		if label == RetryableWriteError {
+			return true
+		}
+	}
+	if wireVersion != nil && wireVersion.Max >= 9 {
+		return false
+	}
+	for _, code := range retryableCodes {
```

Comment:
I think `retryableCodes` is missing `ExceededTimeLimit` (code 262). This was not originally considered a retryable write error, but was added in https://github.com/mongodb/specifications/commit/66631e5b50b8391ead7ea01e21813c450ccbb287#diff-8aaf18b3cf5d8b4260afb96c1517114a.